### PR TITLE
[Feature] Stacked charts (for multi-series) + few charts properties changes

### PR DESCRIPTION
**Pull request by wmai**
⚠️ This branch should not be merged into `main` yet

This PR allows the stacking of charts:

**Separate:**
Screenshot 2022-04-05 at 12 02 28

**Stacked:**
Screenshot 2022-04-05 at 12 02 50

**Stacked (percentage):**
Screenshot 2022-04-05 at 12 02 56

Plus, other small modifications have been made to better match the UI prototypes, such as:

- Legends styles: can be a box, a line or a dashed line
- Source link: has an optional `style` property
- Chart padding can be more precisely defined

### CODE REVIEW

**Review by rkouye — APPROVED** [review]
LGTM !

**Review by richterb — COMMENTED** [review]
Unless I missed something, I didn't see any Story for stacked representations. IMO we need at least one Story for each (chart type, stacking type) relevant combination, to ensure they are subjected to visual testing in Chromatic.

The rest LGTM!

File: packages/visualizations/src/chartjs.d.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import * as chartJS from 'chart.js'; // eslint-disable-line @typescript-eslint/no-unused-vars
+
+// See: https://github.com/y-takey/chartjs-plugin-stacked100/issues/54
```
[the remark this answers is not in the document]
Comment (wmai) [reply packages/visualizations/src/chartjs.d.ts]: It's strange, this piece of code was fixing a typing problem at the time... I just deleted the whole file and the compilation seems to work perfectly, I can't explain what changed 🤷‍♂️.  Good catch 👍

**Comment by richterb** [comment]
🚀 Thanks

**Pull request merged**